tools/tslint-rules/coercionTypesRule.ts

```diff
@@ -2,6 +2,8 @@ import * as ts from 'typescript';
 import * as Lint from 'tslint';
 import * as tsutils from 'tsutils';
 
+const TYPE_ACCEPT_MEMBER_PREFIX = 'ngAcceptInputType_';
```

Comment:
I think I had this hardcoded in one more place. You should be able to replace it with this value as well.

Reply:
yeah. This is already replaced below.